src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -275,9 +276,9 @@ function createCodeCellFromNotebookCell(cell: NotebookCell): nbformat.ICodeCell
     };
 }
 
-function createNotebookCellDataFromRawCell(isNbTrusted: boolean, cell: nbformat.IRawCell): NotebookCellData {
+function createNotebookCellDataFromRawCell(cell: nbformat.IRawCell): NotebookCellData {
     const notebookCellMetadata = new NotebookCellMetadata().with({
-        editable: isNbTrusted,
+        editable: true,
```

Comment:
When the notebook is not editable, then cells are not editable.
This way we don't need to update cell metadata to indicate its editable or not.
Leaving it as always editable is easier (just toggle at the document metatadata level).